src/v3/test/integration/__snapshots__/authenticator-enroll-yubikey-otp.test.tsx.snap

```diff
@@ -122,7 +122,7 @@ exports[`authenticator-enroll-yubikey-otp renders form 1`] = `
                     class="MuiBox-root emotion-15"
                     data-se="icon-yubikey"
                   >
-                    <mocksvgicon />
+                    <undefined />
```

Comment:
Should this be undefined? I expected the actual SVG to be rendered now since its been converted into a function component.

Reply:
this was resolved after converting yubikey svg to function component